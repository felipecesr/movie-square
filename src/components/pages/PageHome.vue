<template>
  <template-home />
</template>

<script>
import TemplateHome from "@templates/TemplateHome.vue";

export default {
  components: { TemplateHome }
};
</script>

<style>
</style>

