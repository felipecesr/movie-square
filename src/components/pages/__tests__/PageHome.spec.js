import { shallowMount } from '@vue/test-utils';
import PageHome from '../PageHome.vue';

describe('PageHome.vue', () => {
  it('renders correctly', () => {
    expect(shallowMount(PageHome).element).toMatchSnapshot();
  });
});
