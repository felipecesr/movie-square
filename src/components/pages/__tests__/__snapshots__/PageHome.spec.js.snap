// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHome.vue renders correctly 1`] = `<home-template-stub />`;
